lib/fsevents-handler.js

```diff
@@ -3,8 +3,34 @@
 var fs = require('fs');
 var sysPath = require('path');
 var readdirp = require('readdirp');
-var fsevents;
-try { fsevents = require('fsevents'); } catch (error) {}
```

Comment:
I am happy to do it like that :)

I only added the function because it was how it is done in `index.js`.